portfolio/src/main/webapp/script.js

```diff
@@ -137,3 +148,141 @@ function deleteSpecificComment(comment) {
   fetch('/delete-given-comment', {method: 'POST', body: params});
 }
 
+/** Creates a map and adds it to the page. */
+async function createMap() {
+  map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 42.4500, lng: -76.4800},
+    zoom: 15.2,
```

Comment:
Rather than declaring a global `map` variable, consider passing this down to the `addMarker()` function as a parameter. Not 100% sure it'll work, but worth a shot!

Reply:
Does work! This is actually what I had originally, but I had changed it because I read some code in the documentation that made the map a global variable. Fixed now